src/plugins/1/essentials/getting-started/index.md

```diff
@@ -20,6 +20,27 @@ A single plugin can implement as many of those interfaces as necessary.
 
 ---
 
+## Plugin Boilerplate
+
+The best way to start developing a plugin is to use one of our boilerplates.
+
+These boilerplates contain a Kuzzle stack specially designed for plugin development.
+
+2 boilerplates are available:
+ - [kuzzle-core-plugin-boilerplate](https://github.com/kuzzleio/kuzzle-core-plugin-boilerplate): contain a simple example of all plugin functionnalities
+ - [kuzzle-plugin-advanced-boilerplate](https://github.com/kuzzleio/kuzzle-plugin-advanced-boilerplate): production boilerplate with better separation of concerns
```

Comment:
... so why should I use the one just before?
I don't really understand the difference between the two: can't we have an overview of all plugin functionalities with a better separation of concerns?
Reading that, it feels that we kept the 1st one for convenience, but the 2nd one is better... And it doesn't feel right.

Reply:
I removed the mention about the second boilerplate for now, I think we need a workshop on this before mentioning it in the documentation